src/helpers/error-message.helper.ts

```diff
@@ -145,9 +145,7 @@ export class ErrorMessageHelper {
         title = `${error.worker}`;
         break;
       case ParseErrorCode.UNKNOWN_VALUE:
-        message = `Nieznana wartość zmiany: "${error.actual}" w dniu ${
-          error.day! + 1
-        }. Przyjęto, że zmiana to wolne.`;
+        message = `Nieznana wartość zmiany: "${error.actual}" w dniu ${error.day! + 1}.`;
```

Comment:
Raczej chodziło mi o to, żeby to przeredagować, a nie usunąć. Tym, co chcemy przekazać jest _Chciałeś mieć X w miejscu Y, ale wpisałeś to źle, więc tego tam __nie ma___ - przydałby się jakiś komentarz, co zrobiliśmy. Myślałam, że może `Nie przypisano zmiany` (co zgadzałoby się z przyjętą konwencją, że zmiany wolne są "puste").

Reply:
Proponuję zrobić to w osobnym zadaniu, ponieważ Aga(nasz grafik) wspominała wczoraj na spotkaniu o ujednoliceniu komunikatów z parsera i z tooltipów.

Reply:
@Ehevi 
Nawet już jest zrobione ;) 
https://app.tara.ai/nurse-scheduling/team-nurse/tasks/386

Reply:
W osobnym głównie dlatego, że potrzeba będzie jeszcze dogadać się z Agą. 
Więc żeby teraz nie męczyć Paulinę

Reply:
ok, nie widziałam tego zadania :stuck_out_tongue: